packages/example-getting-started/README.md

```diff
@@ -1,6 +1,10 @@
 # @loopback/example-getting-started
 
 This is the basic tutorial for getting started with Loopback 4!
+[Click here](https://loopback.io/doc/en/lb4/Getting-started.html) to follow
+along, or jump to the
+[docs section](https://github.com/strongloop/loopback-next/tree/master/packages/example-getting-started/docs)
```

Comment:
+1 to a list linking to the steps that makes it more eye-catching. I'd just saying that it might be better to write down "Section" / "Step" Names instead of names of the `.md` files.

Reply:
The problem I'm currently having is that if I create links internally, then they'll link back to the GitHub repository from within loopback.io, since this whole file is pulled into the docs page as an include.